### PR TITLE
Stomp Smoothing Planning Request Adapter

### PR DESCRIPTION
**Pull request by raghavendersahdev**
### Addition of STOMP Smoothing Adapter in MoveIt

This PR is work done by Raghavender as a part of 2018 Google summer of code. As a part of this PR, a STOMP Smoothing adapter has been implemented. This enables the user to use STOMP as an additional post processing motion planning algorithm in a pipeline with another motion planner like CHOMP, OMPL, etc. This then produces smooth motion plans. 

For tutorials on how to use this motion planning request adapter, refer to PR for a tutorial on Planning Request Adapters  [PR#220](https://github.com/ros-planning/moveit_tutorials/pull/220) . This PR also needs this PR in industrial_moveit package to be merged [PR#78](https://github.com/ros-industrial/industrial_moveit/pull/78) in order for the STOMP smoothing adapter to function properly. 
It should be noted this PR overrides an older PR #965 , after this PR gets merged PR #965 can be closed and ignored.
@davetcoleman @mamoll

### Checklist
- [ ] addition of STOMP smoothing adapter into existing MoveIt Planning Request Adapters
- [ ] merging of the STOMP moveit package from industrial moveit into moveit, this PR overrides this PR #965

### CODE REVIEW

File: moveit.rosinstall [hunk 1]
```diff
@@ -24,3 +24,7 @@
     local-name: srdfdom
     uri: https://github.com/ros-planning/srdfdom.git
     version: kinetic-devel
+- git:
+    local-name: industrial_moveit
+    uri: https://github.com/raghavendersahdev/industrial_moveit.git
```
Comment (davetcoleman) [inline moveit.rosinstall]: this needs to be switched to the new stomp repo that is suppose to be created. fyi @gavanderhoorn

**Comment by rhaschke** [comment]
Please revert the last merge-commit and rebase your branch onto the kinetic-devel to resolve conflicts.
Having merge-commits within a PR-branch like this, makes it impossible to squash, or cherry-pick it to another branch later on.

**Comment by henningkayser** [comment]
@raghavendersahdev what's the current state of this PR? It would be great to get this feature merged. I just think since we have the separate STOMP repo now the smoothing adapter should be added to https://github.com/ros-industrial/stomp_ros instead of here. @gavanderhoorn what do you think?

**Comment by v4hn** [comment]
Obsolete. The plugin is available in https://github.com/ros-industrial/stomp_ros/pull/20

**Pull request closed without merging**